demo/admin/eslint.config.mjs

```diff
@@ -9,6 +9,7 @@ const config = [
     {
         rules: {
             "@calm/react-intl/missing-formatted-message": "off",
```

Comment:
I am not 100% sure if it is a good idea to not internationalize the demo. I would recommand to intern the demo as well.

Reply:
rebased to it.